web/src/lib/roadmap/roadmapBuilder.ts

```diff
@@ -146,6 +147,18 @@ const modifyTasks = (roadmap: RoadmapBuilder, modifications: TaskModification[])
   return roadmap;
 };
 
+const removeDuplicateTasks = (roadmap: RoadmapBuilder): RoadmapBuilder => {
```

Comment:
This was to fix a bug where a non-essential question and a roadmap both add the same task to the roadmap and it would display twice